ledger-core/virtual/object/working_table.go

```diff
@@ -45,11 +43,11 @@ func (wt *WorkingTable) GetResults() map[reference.Global]Summary {
 }
 
 func (wt *WorkingTable) Add(flag contract.InterferenceFlag, ref reference.Global) bool {
```

Comment:
Semantically "Put" fits better here than "Add"

Reply:
From my experience - Put is replace, while add is not replace. Maybe that comes from REST :) I'd prefer leave it as it is